sections/Deals/Vesting/VestingBox/VestingBox.tsx

```diff
@@ -0,0 +1,64 @@
+import { FC, useMemo, useState } from 'react';
+import styled from 'styled-components';
+
+import Connector from 'containers/Connector';
+import TransactionData from 'containers/TransactionData';
+
+import ConfirmTransactionModal from 'components/ConfirmTransactionModal';
+
+import { Container, ContentContainer, ActionButton } from '../../../shared/common';
+
+import { GasLimitEstimate } from 'constants/networks';
+
+interface VestingDealProps {
+	maxValue: number;
```

Comment:
should probably be `vestingAmount` or something a bit more clear

Reply:
Done